server/src-lib/Hasura/RQL/DDL/Schema/Function.hs

```diff
@@ -83,17 +83,17 @@ mkFunctionInfo qf rawFuncInfo = do
   let funcArgsSeq = Seq.fromList funcArgs
       dep = SchemaDependency (SOTable retTable) "table"
       retTable = QualifiedObject retSn (TableName retN)
-  return $ FunctionInfo qf False funTy funcArgsSeq retTable [dep]
+  return $ FunctionInfo qf (SystemDefined False) funTy funcArgsSeq retTable [dep]
```

Comment:
Why we're constructing `SystemDefined` manually? This should come from `HasSystemDefined`!